packages/react/src/lib/accessibility/Behaviors/Popup/popupBehavior.ts

```diff
@@ -16,7 +16,9 @@ const popupBehavior: Accessibility<PopupBehaviorProps> = props => {
   return {
     attributes: {
       trigger: {
-        tabIndex: getAriaAttributeFromProps('tabIndex', props, 0),
+        ...(props.triggerHandlesFocus
```

Comment:
What is the use case of this prop? If this is connected with the `'context'` value on the `on` prop can we use that one instead?

Reply:
it's not only related to context value. I have renamed the prop, hopefully it is more clear now.